lib/server.coffee

```diff
@@ -126,7 +124,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 
 		Spiderable.originalRequest = req
 		url 		= Spiderable._urlForPhantom Meteor.absoluteUrl(), req.url
-		hash 		= SHA256 url
+		hash 		= hashCode url
```

Comment:
Or... Why don't we just use the original url instead... We don't even need the hash.